cards/src/main/java/com/decathlon/vitamin/compose/cards/VitaminCards.kt

```diff
@@ -0,0 +1,258 @@
+     * @param modifier The `Modifier` to be applied to the all card.
+     * @param shape to change the shape of the card.
+     * @param imageWithGradient A boolean value to add a gradient effect to the image. No gradient by default.
+     */
+    @Composable
+    fun FullImage(
+        painter: Painter,
+        contentDescription: String?,
+        content: @Composable (FullImageContent.() -> Unit),
+        modifier: Modifier = Modifier,
+        shape: Shape = RoundedCornerShape(4.dp),
+        imageWithGradient: Boolean = false,
+    ) = Card(modifier = modifier, shape = shape) {
+        Box {
+            CardImage(
+                painter = painter,
+                contentDescription = contentDescription,
+                withGradient = imageWithGradient
+            )
+            Box(modifier = Modifier.align(Alignment.BottomStart).padding(16.dp)) {
+                FullImageContent.content()
+            }
+        }
+    }
+
+    /**
+     * A card with thumbnail image at left
+     * @param painter the [Painter] resource for the image
+     * @param contentDescription to describe image for screen reader. Set to null only if image is decorative.
+     * @param content The content of the card. Can be a composable from [SideImageContent] or a custom composable.
+     * @param modifier The `Modifier` to be applied to the all card.
+     * @param actionContent to add actions icons at the top end of the card like clickable icon.
+     * Can be a composable from [SideImageActionContent] or a custom composable.
+     * @param shape to change the shape of the card.
+     */
+    @Composable
+    fun SideImage(
+        painter: Painter,
+        contentDescription: String?,
+        content: (@Composable SideImageContent.() -> Unit),
+        modifier: Modifier = Modifier,
+        actionContent: @Composable (SideImageActionContent.() -> Unit)? = null,
+        shape: Shape = RoundedCornerShape(4.dp),
+    ) = Card(modifier = modifier, shape = shape) {
+        Column {
+            if (actionContent != null) {
+                SideImageActionContent.actionContent()
+            }
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.spacedBy(16.dp),
+                modifier = Modifier
+                    .padding(
+                        top = if (actionContent != null) {
+                            0.dp
+                        } else {
+                            16.dp
+                        },
+                        bottom = 16.dp,
+                        start = 16.dp,
+                        end = 16.dp
+                    )
+            ) {
+                CardImage(
+                    painter = painter,
+                    contentDescription = contentDescription,
+                    isSideImage = true,
+                    modifier = Modifier.weight(weight = 1f).aspectRatio(ratio = 1f)
```

Comment:
I'm not sure it is a weight here. I think we should use `92.dp` and put it in the size object.